src/main/java/org/spongepowered/common/registry/type/scoreboard/VisibilityRegistryModule.java

```diff
@@ -24,40 +24,30 @@
  */
 package org.spongepowered.common.registry.type.scoreboard;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Maps;
 import net.minecraft.scoreboard.Team;
-import org.spongepowered.api.registry.CatalogRegistryModule;
 import org.spongepowered.api.registry.util.RegisterCatalog;
 import org.spongepowered.api.scoreboard.Visibilities;
 import org.spongepowered.api.scoreboard.Visibility;
+import org.spongepowered.common.registry.type.MutableCatalogRegistryModule;
 
-import java.util.Collection;
 import java.util.Map;
-import java.util.Optional;
 
-public final class VisibilityRegistryModule implements CatalogRegistryModule<Visibility> {
+public final class VisibilityRegistryModule extends MutableCatalogRegistryModule<Visibility> {
```

Comment:
Improved readability, returning a `Set` instead of a `List`, reduction of boilerplate.